### DISCUSSION TITLE
Question about model predictions

### DISCUSSION THREAD

**Discussion by clee758** [talmolab/sleap#1633] [Help!]
I was wondering how predictions of a model are generated? I was observing some of the error data points and their corresponding frame id from the excel sheets from the Jupyter notebook Liezl made for me, but there was no yellow box in the frame of the video, so I couldn't see the extent of the error. Is there a way to see the predictions for that frame?

**Comment by talmo** [comment]
Do you mind resharing that notebook (or the relevant snippets) for context?

Is this for error metric evaluation?

If so, you'll need ground truth (user labeled) data on some frames, after which you can generate predictions on those for subsequent evaluation.

**Reply by clee758** [reply]
Yes, this is for error metric evaluation.

When I go to the corresponding frame_idx in the video, in this case 1122, there is no prediction frame in the SLEAP project, even though there are metrics generated. Why is that? 

![excel1824](https://github.com/talmolab/sleap/assets/117138270/7ae0dd02-c689-4531-a7c8-556b9769a1e1)

**Reply by clee758** [reply]
![Screenshot 2024-01-08 161224](https://github.com/talmolab/sleap/assets/117138270/43f4171a-c39a-42ad-adc4-2aabc4bb28fd)

**Reply by clee758** [reply]
Hi, I'm just checking to see if there is any update regarding the answer?

**Reply by roomrys** [reply]
Hi @clee758,

At least for the example given, the name for the video does not line-up across your slp file and the video name(s) listed in your spreadsheet:
| csv or slp | video name |
|----------|----------|
| csv | ...0230407.mp4 |
| slp | ...0230419_1.mp4 |

Are you sure you are looking at the correct video in the correct slp project?

Thanks,
LIezl
